metrics/prometheus.go

```diff
@@ -1565,8 +1565,8 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 				},
 			},
 			{
-				name:        "container_perf_metric_scaling_ratio",
-				help:        "Perf event metric scaling ratio",
+				name:        "container_perf_events_total_scaling_ratio",
```

Comment:
this shouldn't include `total`.  `total` indicates that the metric is cumulative, which I don't believe this is.  `total` should only be placed at the end of the metric when appropriate.